src/utils/analytics.ts

```diff
@@ -0,0 +1,20 @@
+interface AnalyticsEvent {
+  event_name: string;
+  event_category: string;
+  event_label?: string;
```

Comment:
I would make the difference between these three clearer here. I'm not sure when an event would want to use `event_category` vs. `event_label`.